packages/mask/src/web3/UI/EthereumChainBoundary.tsx

```diff
@@ -204,7 +199,6 @@ export function EthereumChainBoundary(props: EthereumChainBoundaryProps) {
                 ) : null}
                 {isAllowed ? (
                     <ActionButtonPromise
```

Comment:
How to set the size of button? I think we should pass `props.size`

Reply:
`ActionButtonPromiseProps`